### DISCUSSION TITLE
[Feature Request] Can the "Application" options also process flags beginning with `-` please

### DISCUSSION THREAD

**Discussion by Smurf-IV** [pocoproject/poco#4281] [General]
Most command line processors accept **both** types of flag denominators, i.e. `-` and `/` (or `\`)

Please add the `-` option; regardless of Unix / windows / etc OS style

**Comment by andrewauclair** [comment]
Are you looking for `void setUnixOptions(bool flag);` which is already in the `Application` class?

It's false by default on Windows. Just need to follow the instructions in the comment for `setUnixOptions` to change it.

**Comment by Smurf-IV** [comment]
Thanks for the reply.
Still: most "Windows" command line apps accept both styles, 
hence; I request for a documentation update, if the code is not going to change ;-)
